packages/synapse-react-client/src/components/Markdown/MarkdownSynapse.test.tsx

```diff
@@ -1,22 +1,25 @@
 import { render, screen, waitFor, within } from '@testing-library/react'
```

Comment:
Changed a bunch of tests and stories -- for all of them, I fixed type + lint errors and moved them inside of `synapse-react-client/src`.

Now we have no more non-linted/non type-safe tests!